tests/sub_group_mask/sub_group_mask_extract_bits.cpp

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests to check sub_group_mask extract_bits()
+//
+*******************************************************************************/
+
+#include "sub_group_mask_common.h"
+
+#define TEST_NAME sub_group_mask_extract_bits
+
+namespace TEST_NAMESPACE {
+
+using namespace sycl_cts;
+#ifdef SYCL_EXT_ONEAPI_SUB_GROUP_MASK
+
+template <typename T>
```

Comment:
Add a comment explaining the magic in English

Reply:
Added comment here https://github.com/KhronosGroup/SYCL-CTS/pull/202/commits/9151ab676d35a569cb0c5a89104139b55590d9b5